### PR TITLE
Add highlight

### PR DESCRIPTION
**Pull request by SinceJune**

### CODE REVIEW

**Comment by SinceJune** [comment]
see pictures:
image
image

File: src/main/java/io/dashbase/es/queries/QueryConverter.java [hunk 1]
```diff
@@ -26,7 +32,17 @@ public EsQueryResponse rapidResToEsRes(Map originRes, RapidResponse rapidRes, bo
         esRes.took = rapidRes.latencyInMillis;
         esRes.timed_out = rapidRes.isTimedOut;
         String index = String.join(",", rapidRes.request.tableNames);
-        esRes.hits.hits = rapidRes.hits.stream().map(hit -> rapidHit2EsHit(hit, index, sourceRetrieval)).collect(Collectors.toList());
+
+        //highlight support
+        Map<String, Object> highLight = (Map<String, Object>) originRes.get("highlight");
```
Comment (javasoze) [inline src/main/java/io/dashbase/es/queries/QueryConverter.java]: add nullcheck, highlight maynot be requested and maybe null

File: src/main/java/io/dashbase/es/queries/QueryConverter.java [hunk 2]
```diff
@@ -26,7 +33,21 @@ public EsQueryResponse rapidResToEsRes(Map originRes, RapidResponse rapidRes, bo
...
         String index = String.join(",", rapidRes.request.tableNames);
-        esRes.hits.hits = rapidRes.hits.stream().map(hit -> rapidHit2EsHit(hit, index, sourceRetrieval)).collect(Collectors.toList());
+
+        //highlight support
+        ObjectMapper objectMapper = new ObjectMapper();
+        EsHighlight esHighlight = new EsHighlight();
+        try {
+            esHighlight = objectMapper.convertValue(originRes.getOrDefault("highlight", new HashMap<String, Object>()), EsHighlight.class);
+        } catch (IllegalFormatException e) {
+            e.printStackTrace();
```
Comment (javasoze) [inline src/main/java/io/dashbase/es/queries/QueryConverter.java]: avoid e.printStackTrace, always use logger.error()

**Comment by javasoze** [comment]
@SinceJune I loaded Kibana with this PR, searching for: ```apple``` does not show highlights. Looks like highlighting is own shown on meta fields?

**Pull request merged**